### PR TITLE
Adds article back

### PR DESCRIPTION
**Pull request by Rafase282**
Adds the article that was deleted on #584 back with a better name and
without the unrelated material

### CODE REVIEW

File: Programing-Paradigm-And-Methodology.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Programing Paradigm and Methodology
+## Foundational Agile Principles
+**_Individuals and interactions_** over processes and tools
```
Comment (abhisekp) [inline Programing-Paradigm-And-Methodology.md]: - reformat using triple asterisks to bold italic
- bold italic like the following

```md
***Individuals and interactions*** over ***processes and tools***
```
> ***Individuals and interactions*** over ***processes and tools***

Comment (abhisekp) [inline Programing-Paradigm-And-Methodology.md]: Reformat using proper markdown using triple asterisks to ***bold and italics***.

```md
***Individuals and interactions*** over ***Processes and tools***
```

> ***Individuals and interactions*** over ***Processes and tools***

File: Programing-Paradigm-And-Methodology.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+**_Working software _**over comprehensive
+
+_**Customer collaboration**_ over contract negotiation
+
+_**Responding to change **_ over following a plan
+
+## User-Stories
+User stories help us tie the needs of our users directly to features we implement in a conversational format. They **always** follow this syntax: _"As a User/Stakeholder , I need/want to be able to Do Something"_. This can result in some awkward stories for the "obvious" features like _"As a user, I need to be able to trust that my credit card information is secure."_, but it is imperative that ALL features tie back to a story in order to allow us to effectively break the work down into manageable pieces.
```
Comment (abhisekp) [inline Programing-Paradigm-And-Methodology.md]: *...User/Stakeholder , I need/want to be ab...*

the comma should be attached with *Stakeholder,*

File: Programing-Paradigm-And-Methodology.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Programing Paradigm and Methodology
```
Comment (abhisekp) [inline Programing-Paradigm-And-Methodology.md]: This is just 

# Programing Methodology

**Comment by abhisekp** [comment]
@Rafase282 Add the link to http://scrummethodology.com at the end.

**Comment by abhisekp** [comment]
Although more explanations are needed but someone else could do that afterwords (*optional*).

File: Programing-Methodology.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Programing Methodology
+
+## Foundational Agile Principles
+
+***_Individuals and interactions_*** over **_Processes and tools_**
```
Comment (abhisekp) [inline Programing-Methodology.md]: Fix markdown

File: Programing-Methodology.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+## Sprints
+A "Sprint" is a short (1-3 weeks) development cycle during which a number of stories or subtasks are targeted for completion by the end of the sprint. The idea behind "Sprinting" is to allow us to **deliver** a story to the project champion for feedback/approval. This iterative workflow ensures we get continuous buy-in from the project champion and we are delivering functionality that both works, and is valuable.
+
+## SCRUM
+At a basic level, SCRUM helps us stay focused and informed about what the status is on each assigned task as well as break down communication barriers. In many professional settings the work day begins with a "Daily Scrum".  We will be following a weekly scrum to align with our sprints.  A common scrum failure is to attempt to problem solve during a scrum. Troubleshooting and brainstorming should always be done as a separate task from SCRUM.
+Read more [here.](http://scrummethodology.com/)
```
Comment (abhisekp) [inline Programing-Methodology.md]: Add this after a newline

**Comment by abhisekp** [comment]
# :+1: LGTM

File: Programing-Methodology.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Programing Methodology
```
Comment (abhisekp) [inline Programing-Methodology.md]: typo: `Programing` -> `Programming`

**Comment by abhisekp** [comment]
Typo in filename `Programing-Methodology.md` -> `Programming-Methodology.md`

**Comment by Rafase282** [comment]
Done.

**Comment by abhisekp** [comment]
# :+1: LGTM

**Pull request merged**